src/Nav/stories/mockData.js

```diff
@@ -15,7 +15,49 @@ const mockData = [
     pages: [
       {
         importedFileName: 'webhook-docs-intro',
-        pages: [],
+        pages: [
+          {
+            importedFileName: 'webhook-docs-intro',
```

Comment:
Do we actually need the `importedFileName`? 
Last time I checked it in the manifest parser this field was not being used, but I might have missed some parts of the code.

Reply:
No, it's just an artifact of the legacy format.